packages/contracts/deploy/new/30_plugins/10_plugin-repos/30_create_multisig_repo.ts

```diff
@@ -49,3 +50,4 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
 export default func;
 func.tags = ['New', 'CreateMultisigRepo'];
+func.skip = skipDeploy;
```

Comment:
Is the idea that all plugins are skipped by default? Also don't we need the multisig plugin to deploy the management dao

Reply:
Not for this specific deployment we're doing. The plugin installation will be handled separately.

Reply:
Ah, so the idea is that this is just a temporary solution since all these files are anyway deleted in https://github.com/aragon/osx/pull/551 ?

We shouldn't forget to delete `skipDeploy` then.